.github/pipeline-descriptor.yml

```diff
@@ -0,0 +1,30 @@
+github:
+  username: ${{ secrets.JAVA_GITHUB_USERNAME }}
+  token:    ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
+
+codeowners:
+- path:  "*"
+  owner: "@paketo-buildpacks/java-maintainers"
+
+package:
+  repositories:   ["docker.io/paketobuildpacks/pipeline-builder-stool-pigeon","gcr.io/paketo-buildpacks/pipeline-builder-stool-pigeon"]
+  register:       true
```

Comment:
Do you know if the buildpacks registry supports extensions? When this is true, we attempt to register the buildpack with the registry. Maybe we should keep this as false till we can confirm?

Reply:
Interesting question.. I'll disable it via another pipeline-builder PR for now, and raise the question in one of the workgroup meetings.. eg, do we even want a registry for extensions?

Reply:
Thanks. I did post https://cloud-native.slack.com/archives/C032YE21V1T/p1689728506543929 but no response yet.

Reply:
In the commit below, I've set it to false in this projects pipeline-descriptor, and within 
https://github.com/paketo-buildpacks/pipeline-builder/pull/1219
I've added to the test to ensure registration will only occur for buildpacks. 
The pipelines in the commit below were regenned with that change in place, and you see the result at line 247 of pb-create-package where the if statement now has false && false ( registration-requested-in-pipeline-descriptor && !extension )

Reply:
👍 I just merged your PR. Does this PR need an update to the go.mod, or are we good to go?

Reply:
this project doesn't depend on pipeline-builder in go.mod, and it's pipelines were generated with octo at the commit level you just merged =) so we should be good =)